composite/market-closure/README.md

```diff
@@ -16,8 +16,7 @@ The adapter takes the following environment variables:
 | :-------: | :-----------------------------------------: | :-------------------------------------------: | :------------------------: | :---------: |
 |    ✅     |                `CHECK_TYPE`                 |   The provider to check if a market is open   | `schedule`, `tradinghours` |             |
 |    🟡     | `CHECK_API_KEY` (when using `tradinghours`) |  An API key when needed by a check provider   |                            |             |
-|    ✅     |               `PRICE_ADAPTER`               | The provider to use for retrieving price data |    `finnhub`, `fcsapi`     |             |
-|    🟡     |                  `API_KEY`                  |  An API key when needed by a price provider   |                            |             |
+|    ✅     |             `DATA_PROVIDER_URL`             | The adapter URL for the price data provider   | `finnhub`, `fcsapi`        |             |
```

Comment:
Nitpicking, but I find this confusing. I foresee feeds team reading this and setting `DATA_PROVIDER_URL = finnhub` instead of `FINNHUB_DATA_PROVIDER_URL=https://whatever`

Reply:
I've made an attempt to make the documentation more clear. Please see if it makes more sense now